public/graph.js

```diff
@@ -499,6 +530,36 @@ class GraphController {
                 )
                 .attr("width", Width / Object.keys(dailyReceivedTotal).length);
 
+            let tip;
+            receivedLayer
+                .selectAll("rect")
+                .on("mouseover", (d, i, n) => {
+                    // Get color of hovered rect
+                    let rgb = d3.select(n[i]).style("fill")
+                    // Get color components from an rgb string
+                    rgb = rgb.substring(4, rgb.length-1).replace(/ /g, '').split(',');
+                    // Cast strings in array to int
+                    rgb = rgb.map((x) =>parseInt(x));
+                    let hex = rgbToHex(...rgb)
```

Comment:
If operator colors are already stored in hex format, we should probably pull this from the table instead?